Describe the limitations of the Sortable options (#933)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/GridBase/1 Configuration/rowDragging/allowReordering.md

Allows row reordering using drag and drop gestures.

---

> This option allows users to reorder rows only in the UI. You should also handle drag and drop in code. If users reorder rows inside a single component, implement the [onReorder]({basewidgetpath}/Configuration/rowDragging/#onReorder) handler. 

Reviewer: I am not clear how the 1st and 2nd sentence here relate to each other. Is something missing? These ideas don't seem to flow as written.

Author: When users set the allowReordering option to true, they can drag rows in the following manner:
![image](https://user-images.githubusercontent.com/33040934/85848803-77c9d880-b7b2-11ea-9699-0d0a867ea7b9.png)
But it has only visual effect. The widget's data is not modified, and after a row is dropped, the widget will look the same as before the user started dragging:
![image](https://user-images.githubusercontent.com/33040934/85861567-3f81c480-b7c9-11ea-8680-25630468263a.png)
 We need to modify the the widget's data source in code, and this is the reason why I contrast reorder in UI and in code (having in mind that reorder and drag and drop here mean the same thing). Only when we implement the onReorder handler and then drag a row, the row order will change both in code and in the UI:
![image](https://user-images.githubusercontent.com/33040934/85861010-5247c980-b7c8-11ea-99fa-58922eb701b4.png)

Reviewer: This option limits row reordering to the UI and does not affect the data source. You should also handle the effects of the drag and drop to the data source in code. If users reorder rows inside a single component, implement the [onReorder]({basewidgetpath}/Configuration/rowDragging/#onReorder) handler.

> To configure drag and drop between components, follow the instructions from the [group]({basewidgetpath}/Configuration/rowDragging/#group) topic.

Reviewer 2: Since the `group` article doesn't really provide any instructions:

```suggestion
For information on drag and drop between components, refer to the [group]({basewidgetpath}/Configuration/rowDragging/#group) topic.
```

Allows row reordering using drag and drop gestures.

---
This option requires that also you implement the [onReorder]({basewidgetpath}/Configuration/rowDragging/#onReorder) handler to allow users to reorder rows inside a single component. 

To perform cross-component drag and drop, implement at least one of the following handlers instead of the **onReorder**:     

- [onRemove]({basewidgetpath}/Configuration/rowDragging/#onRemove) to handle start of item dragging from one component 
- [onAdd]({basewidgetpath}/Configuration/rowDragging/#onAdd) to handle item drop to another component

> If you need to forbid users to drag items inside or between the components, specify the rules in the [onDragChange]({basewidgetpath}/Configuration/rowDragging/#onDragChange) handler. 

Reviewer 2: I went to the onDragChange description and didn't found any further information on the use case. Furthermore, I believe that to forbid users to drg items, I can simply keep allowReordering disabled, why bother with onDragChange?

I also had a look at the [Drag & Drop Between Grids](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/DnDBetweenGrids/Angular/Light/) demo, and it looks like allowReordering has nothing to do with cross-component drag and drop. Why provide so many details on this use case here?

Author: 1. My description was not certain enough. It should be "If you need to forbid users to drag items inside or between the **certain** components, specify the rules..." in the same way as it is described in the  [Node d&d demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/LocalReordering/jQuery/Light/). I also think that I should add a snippet on the onDragChange description.

2. Previously, I just haven't found a better topic to place this info and followed the task as it was written. Alyar suggested moving this to the [group](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#group) option. What do you think?
